docs/pages/build-reference/infrastructure.md

```diff
@@ -30,7 +30,7 @@ When selecting an image for the build you can use the full name provided below o
 - Android workers run on Kubernetes in an isolated environment
   - Every build gets its own container running on a dedicated Kubernetes node
   - Build resources: 4 CPU, 12 GB RAM
-- NPM cache deployed with Kubernetes. [Learn more](caching/#javascript-dependencies)
+- npm cache deployed with Kubernetes. [Learn more](caching/#javascript-dependencies)
```

Comment:
```suggestion
- [npm cache deployed with Kubernetes](caching/#javascript-dependencies)
```

I think we can provide the link directly to the subject rather than adding a "Learn more".